Mention pixi in installation guide (#3678)
ManimCommunity/manim: docs/source/installation.rst

.. _conda-installation:

> Installing Manim in conda or pixi
> *********************************

Reviewer: ```suggestion
Installing Manim via Conda and related environment managers
***********************************************************
```

> Conda and pixi are package managers for Python that allows creating environments

Reviewer: ```suggestion
Conda is a package manager for Python that allows creating environments
```
